### ISSUE TITLE
Feature: toggle on click

### ISSUE THREAD

**Issue by lostmsu** [ageitgey/show-facebook-computer-vision-tags#17]
Some pages have mini-pictures. Overlay from this extension makes them hard to view. Would be nice to make them togglable.

**Comment by J2TEAM** [comment]
Agree! But why are we not using **hover** instead of **click**? :smile:

@ageitgey  What do you think?
